app/src/main/java/org/kiwix/kiwixmobile/utils/AnimationUtils.java

```diff
@@ -61,4 +65,30 @@ public boolean willChangeBounds() {
         (int) (initialHeight / view.getContext().getResources().getDisplayMetrics().density));
     view.startAnimation(animation);
   }
+
+  //rotate animation for closeAllTab FAB
+  public static void rotate(FloatingActionButton v) {
+    float mAngleToRotate = 360f; // rotate 1 round
```

Comment:
Don't use the Hungarian notations checkout our code style guidelines in the repo.